src/audiomixerboard.cpp

```diff
@@ -1305,6 +1307,49 @@ void CAudioMixerBoard::SetAllFaderLevelsToNewClientLevel()
     }
 }
 
+void CAudioMixerBoard::AutoAdjustAllFaderLevels()
+{
+    QMutexLocker locker ( &Mutex );
+
+    for ( int i = 0; i < MAX_NUM_CHANNELS; ++i )
+    {
+        // only apply to visible faders (and not to my own channel fader)
+        if ( vecpChanFader[i]->IsVisible() && ( true /* i != iMyChannelID */ ) )
+        {
+            // get averaged level meter value (range 0...8)
+            float levelMeter = vecAvgLevels[i];
```

Comment:
This appears to be referenced only once.  I'd rather it didn't exist and `vecAvgLevels[i]` was referenced directly - it makes the code easier to follow.